local-volume/provisioner/pkg/common/common.go

```diff
@@ -36,11 +37,17 @@ const (
 	// NodeLabelKey is the label key that this provisioner uses for PV node affinity
 	// hostname is not the best choice, but it's what pod and node affinity also use
 	NodeLabelKey = apis.LabelHostname
+	// VolumeTypeFile represents file type volumes
+	VolumeTypeFile = "file"
+	// VolumeTypeBlock represents block type volumes
+	VolumeTypeBlock = "block"
 
 	// DefaultHostDir is the default host dir to discover local volumes.
 	DefaultHostDir = "/mnt/disks"
 	// DefaultMountDir is the container mount point for the default host dir.
 	DefaultMountDir = "/local-disks"
+	// DefaultVolumeType is default type of volume being discovered in the specified host dir.
+	DefaultVolumeType = VolumeTypeFile
```

Comment:
@ianchakeres GetDefaultVolumeConfig() is only invoked if no configmap has been supplied. This default value is used when a configmap has been used and a volumeType has not been specified. It basically indicates that the default setting for volumetype when not specified is file.